tensorboard/webapp/plugins/npmi/actions/npmi_actions.ts

```diff
@@ -122,7 +122,21 @@ export const npmiShowHiddenAnnotationsToggled = createAction(
   '[NPMI] Show Hidden Annotations Toggled'
 );
 
+export const npmiToggleEmbeddingsView = createAction(
```

Comment:
Prefer past tense for these action names, i.e., `npmiEmbeddingsViewToggled`. Here and below.